packages/api-catalog-service/src/datasources/namespaceDB.ts

```diff
@@ -0,0 +1,158 @@
+import { MongoDataSource } from 'apollo-datasource-mongodb';
+import { deepMergeByKey } from 'utils';
+
+import { NamespaceModel } from 'db/namespace';
+
+import {
+  ApiCategoryCountArg,
+  CreateNsDocArg,
+  INameSpaceModelDoc,
+  ListNamespaceArg,
+  UpdateNsDocArg,
+} from './types';
+
+export class NamespaceDB extends MongoDataSource<NamespaceModel, IContext> {
+  async getANamespaceById(id: string) {
+    return this.findOneById(id);
+  }
+
+  async getANamespaceBySlug(slug: string) {
+    const namespace = await this.model.findOne({ slug });
+    JSON.stringify(namespace);
```

Comment:
JSON.stringify operation seems to be unused programatically

Reply:
removed